cmd/minishift/cmd/start.go

```diff
@@ -274,11 +291,24 @@ func startHost(libMachineClient *libmachine.Client) (*host.Host, string) {
 		ShellProxyEnv:    shellProxyEnv,
 	}
 
-	fmt.Printf("Starting local OpenShift cluster using '%s' hypervisor...\n", machineConfig.VMDriver)
+	fmt.Printf(" using '%s' hypervisor ...\n", machineConfig.VMDriver)
 	var hostVm *host.Host
+
+	// configuration with these settings only happen on create
+	isRestart := cmdutil.VMExists(libMachineClient, constants.MachineName)
+	if !isRestart {
+		fmt.Println("-- Instance will be configured with ...")
+		fmt.Println("   Memory:   ", units.HumanSize(float64((machineConfig.Memory/units.KiB)*units.GB)))
+		fmt.Println("   vCPUs :   ", machineConfig.CPUs)
+		fmt.Println("   Disksize: ", units.HumanSize(float64(machineConfig.DiskSize*units.MB)))
+		fmt.Println("   Boot ISO: ", machineConfig.MinikubeISO)
```

Comment:
> Agree with @hferentschik . It should show the local cached version

Right, since in the end it is always an ISO somewhere on disk. That said, I think it would take quite some doing to get this right. There are several things which needs addressing. I'd prefer to remove this and create a follow up issue where we can take care of some ISO caching related things (for example also adding the ISO version into the cache path). There are a lot of nice things in this pull request and I'd prefer to get them merged asap.